org.adempiere.base/src/org/compiere/model/MUserDefWin.java

```diff
@@ -104,26 +105,25 @@ public MUserDefWin(Properties ctx, MUserDefWin copy, String trxName)
 	 */
 	private static MUserDefWin[] getAll (Properties ctx, int window_ID )
 	{
-		synchronized (m_fullListLock) {
-			if (m_fullList == null) {
-				try {
-					PO.setCrossTenantSafe();
-					m_fullList = new Query(ctx, MUserDefWin.Table_Name, null, null)
-							.setOnlyActiveRecords(true)
-							.list();
-				} finally {
-					PO.clearCrossTenantSafe();
-				}
+		List<MUserDefWin> fullList = null;
```

Comment:
tested this change successfully with two tenants both customizing Sales Order window